R/init.R

```diff
@@ -153,6 +153,16 @@ init <- function(data,
   # log
   teal.logger::log_system_info()
 
+  if (!identical(Sys.getenv("TESTTHAT"), "true")) {
+    future::plan(future::multisession, workers = 2)
+    logger::log_trace("future::plan() set: using future::multisession and 2 workers.")
+    lockfile_task <- ExtendedTask$new(create_lockfile_future)
+    lockfile_task$invoke()
+    logger::log_trace("pak::lockfile_create() has been started in a parallel process through shiny::ExtendedTask$new().")
```

Comment:
I guess you don't have to put implementation details in the log message. Something like "lockfile process started" should be enough.

Reply:
agree